tests/test_recompute_navmesh.py

```diff
@@ -69,31 +72,43 @@ def test_recompute_navmesh(test_scene, sim):
     assert sim.recompute_navmesh(sim.pathfinder, navmesh_settings)
     assert sim.pathfinder.is_loaded
 
+    # get the re-computed navmesh area
+    recomputedNavMeshArea1 = sim.pathfinder.navigable_area
+    assert loadedNavMeshArea == recomputedNavMeshArea1
```

Comment:
This fails for skokloster castle scene because recomputed navmesh with default params includes the roof as navigable. We must generated that NavMesh with special considerations. 

Thoughts on resolution for this? @erikwijmans 

We could test this differently.

Reply:
In terms of testing, this is really testing the recompute logic, not the area logic (i.e. as long as the area logic is deterministic, it can be broken and still pass this test).  Comparing to golden values is really the only thing I can think of to test regressions.  In terms of testing correctness in the first place, that would either need manual inspection to see if the area values are sensible or testing against some mesh where we can derive the area manually.

Reply:
Good point, still curious what we did to remove the ceiling here. This is relevant for analogous runtime re-compute during training.

I'll run some validation tests and then include golden values for the test scenes. 👍

Reply:
Updated testing sections of description for this.